WTWSMS/common/dynasties/erik_dynasties.txt

```diff
@@ -77,4 +77,14 @@
 			
 	}
 
+}
+1100004 = { # for made-up Gallus, governor of Thrace
+	name="Gallus"
+	culture = thraco_roman
+	religion = latin_solar_pagan_reformed
+}
+11000111 = { # for made-up Eucleidas, governor of Thrace
+	name="Eucleidas"
+	culture = tsakonian
+	religion = mithraic_pagan
```

Comment:
Why did you add these here?

Reply:
Eucleidas is in zusk characters, so I thought it's a fitting place

Reply:
Yes, I understand. But isn't the intent to have random dynasties?

Reply:
Euclidas has a defined dynasty ID, but that dynasty was missing, so I added it

Reply:
This is Eucleidas:
11000111 = {
	name="Eucleidas"
	# Make-up Governor of Thrace
	religion=mithraic_pagan
	culture=tsakonian
	dynasty=11000111
	414.1.1={
		birth = yes
	}
	464.1.1={
		death = yes
	}
}

Reply:
Yes, but isn't the effect to get a random one?

Reply:
Will check in game.

Reply:
An Eucleidasid dynasty is generated properly, but is the "dynasty=" line required to generate?

Reply:
I don't know, but you can test.

Reply:
Tested, it's not needed to generate a dynasty. But given that this character is in 395-476 period and he may get a family in the future, I still think defining a dynasty is a better solution.